### PR TITLE
added pip, python-dev and gcc requirements

### PR DESCRIPTION
**Pull request by kobiam**
Hi, please review it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,19 @@ This repository contains Cloudify's system tests which in most cases mean that t
 
 In addition, plugins functionality is tested and Cloudify's examples.
 
+* requirements:
```
Comment (nir0s) [inline README.md]: Should be # Requirements, no?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,19 @@ This repository contains Cloudify's system tests which in most cases mean that t
...
+
+pip version 1.5 or higher.
```
Comment (nir0s) [inline README.md]: this should be a list of requirements

File: README.md [hunk 3]
```diff
@@ -15,6 +15,19 @@ This repository contains Cloudify's system tests which in most cases mean that t
...
+python-devel (centos).
+python-dev (ubuntu).
+gcc (ubuntu, centos).
+
+yum install python-devel gcc -y
+apt-get install python-dev gcc -y
+
+add server_name to the manager-inputs file.
```
Comment (nir0s) [inline README.md]: I don't understand what I need to do from this and it's not `server_name`, it's `manager_server_name` as far as I know.

File: README.md [hunk 4]
```diff
@@ -15,6 +15,19 @@ This repository contains Cloudify's system tests which in most cases mean that t
 
 In addition, plugins functionality is tested and Cloudify's examples.
 
+* requirements:
+
+pip version 1.5 or higher.
+python-devel (centos).
+python-dev (ubuntu).
+gcc (ubuntu, centos).
```
Comment (nir0s) [inline README.md]: gcc is required on any distribution, not just ubuntu and centos

File: README.md (hunk 3 above)
Comment (kobiam) [inline README.md]: please check the second commit

**Comment by geokala** [comment]
Closed: As this PR is more than six months old without having been merged. Please reopen (and rebase if necessary) if it is still relevant.

**Pull request closed without merging**